locales/en/plugin__odf-console.json

```
@@ -37,6 +37,8 @@
   "Clusters": "Clusters",
   "Applications": "Applications",
   "DRPolicies": "DRPolicies",
+  "No disaster recovery policies yet": "No disaster recovery policies yet",
```

The CR name is DR policy. so we should use that. Also aligning with what we have in console as a not found text.
```suggestion
  "No DRPolicy found": "No disaster recovery policies yet",
```

---

UX does not agree to keep it like this, they said it is similar to ACM's empty page message. so they want to keep it like this.